content/running_bookwyrm/monitoring-queue.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: Monitoring Queue
+Date: 2022-11-23
+Order: 6
+---
+
+There might be occurences where your instances behaves slow. One option would
+be to inspect the queue to see, whether some jobs hang. Read on to learn how.
+
+## Celery
+
+BookWyrm is using [Celery](https://docs.celeryq.dev/en/stable/) to manage
+background jobs.
+
+## Flower
+
+To watch Celery jobs in real-time BookWyrm uses
+[Flower](https://flower.readthedocs.io/en/latest/).
+
+In case you haven't tweaked the
+[`docker-compose.yml`](https://github.com/bookwyrm-social/bookwyrm/blob/dc14670a2ca7553317528d3384146d79df1f7413/docker-compose.yml#L87-L100) you can find the
+service on [port 8888](https://github.com/bookwyrm-social/bookwyrm/blob/dc14670a2ca7553317528d3384146d79df1f7413/.env.example#L42-L45).
```

Comment:
This is just me (maybe others can disagree) but I like to see the whole url. So explicitly saying something like http://yoururl.com:8888. In my experience I also used https instead of http and was wondering why it wasn't working... So it might be good to point out.

Reply:
Good point. I took inspiration from the external storage page and added a line.
Thanks for the feedback.